From ca2909a5657ab4134c80989e510575127c224188 Mon Sep 17 00:00:00 2001
From: Chris Wailes <chriswailes@google.com>
Date: Mon, 1 Feb 2021 14:50:43 -0800
Subject: [PATCH 1/3] Added new target to annotation library visibility

This CL adds artservice to the visibility list for
art.module.api.annotations.for.system.modules.

Test: m com.android.art
Test: art/build/apex/runtests.sh
Bug: 177273468
Merged-In: I580b41fa183cf077d7fefea02350b2d324cf6a22
Change-Id: I580b41fa183cf077d7fefea02350b2d324cf6a22
---
 JavaLibrary.bp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/JavaLibrary.bp b/JavaLibrary.bp
index 92bbc6692e..ff82248452 100644
--- a/JavaLibrary.bp
+++ b/JavaLibrary.bp
@@ -1019,6 +1019,9 @@ java_system_modules {
 // avoid introducing compile time cycles.
 java_library {
     name: "art.module.api.annotations.for.system.modules",
+    visibility: [
+        "//art/libartservice",
+    ],
     srcs: [
         ":api_surface_annotation_files",
     ],
-- 
2.30.2

